Labs/Lab2 - VM Creation/Lab2.a - Create a Windows Virtual Machine/README.md

```diff
@@ -44,10 +44,11 @@ At this point you will be taken to the Virtual Machine service with several blad
 1. Select the Region you are in from the dropdown menu
 1. Under Availability options, select **No infrastructure redundancy required**
 1. Select **Standard** Security type
-1. For Image, click on See all images and select **Windows Server 2019 Datacenter - Gen1**
+1. For Image, click on See all images and select **Windows Server 2019 Datacenter - Gen2**
```

Comment:
Any particular reason to go with gen2? I wanted to keep it gen1 considering we're only running small scripts there.